src/components/SiteList.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import summary from '../apis/summary';
+
+import SiteItem from './SiteItem';
+
+class SiteList extends Component {
+  state = {
+    sites: []
+  };
+
+  componentDidMount() {
+    summary.get('/sites').then(response => {
+      this.setState({ sites: response.data.features });
+    });
+  }
+
+  siteSelectHandler(id) {
+    this.props.history.push({ pathname: '/' + id });
+  }
+
+  render() {
+    let sites = this.state.sites.map(site => {
+      return (
+        <div className="item" key={site.id}>
+          <div className="header">
+            <SiteItem site={site} clicked={() => this.siteSelectHandler(site.id)} />
```

Comment:
I would suggest explicitly adding the `Fn`-suffiix to names of props that are functions. Choosing the right name helps other developers with clarity!

Reply:
no problem